packages/discovery-react-components/package.json

```diff
@@ -15,7 +15,7 @@
     "circular": "madge --circular src/*",
     "build": "rollup -c",
     "eslint": "eslint --quiet '{src,.storybook}/**/*.{js,jsx,ts,tsx}'",
-    "lint": "yarn run eslint && yarn run circular",
+    "lint": "yarn run circular && yarn run eslint",
```

Comment:
Not asking to revert this, just curious if there was a reason these needed to be swapped.

Reply:
our husky config passes `--fix` along to `yarn lint`. `circular` must have been ignoring extra parameters before. now it gets upset. this allows `--fix` to properly be passed to `eslint` instead